crates/biome_js_analyze/tests/specs/nursery/useIncludes/invalid.js

```diff
@@ -0,0 +1,12 @@
+// should generate diagnostics
+
+"foo".indexOf("o") !== -1;
+["a", "b", "c"].indexOf("a") === -1;
+
+"foo".indexOf(bar) >= 0;
+
+["a", "b", "c"].indexOf(bar) < 0;
+
+/a/.test("abc");
```

Comment:
The code fix is quite complex and I'd like to see more tests for it. Does it work in `if`/`while`? What about when it's part of more complex conditions eg `"foo".indexOf("o") !== -1 && bar > 4`? Do we preserve comments?

Feel free to make additional `invalid` files if that helps stay organized.